controllers/nutanixcluster_controller.go

```diff
@@ -152,10 +154,10 @@ func (r *NutanixClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	}
 	conditions.MarkTrue(cluster, infrav1.CredentialRefSecretOwnerSetCondition)
 
-	client, err := CreateNutanixClient(ctx, r.SecretInformer, cluster)
+	v3Client, err := CreateNutanixClient(r.SecretInformer, r.ConfigMapInformer, cluster)
```

Comment:
Don't we need a "reconcileAdditionalTrustBundle" here?

Reply:
just checking, do we want to support users to update this additionalbundle later

Reply:
The trust bundle only needs to be updated when the root CA used to issue the certificate has changed. Wondering if it's often enough to warrant handling with a reconciler.

Reply:
lets track it separately